src/react/account/AccountPoliciesList.tsx

```diff
@@ -0,0 +1,234 @@
+  const IAMClient = useIAMClient();
+  const [showModal, setShowModal] = useState(false);
+  const { status: listPoliciesStatus } = useQuery(getPoliciesQuery(policyName, notFalsyTypeGuard(IAMClient)));
+  const deletePolicyMutation = useMutation(
+    (policyName: string) => {
+      return notFalsyTypeGuard(IAMClient)
+        .deletePolicy(accountName, policyName);
+    },
+    {
+      onSuccess: () => queryClient.invalidateQueries(getPoliciesQuery(accountName, notFalsyTypeGuard(IAMClient)).queryKey),
+    },
+  );
+
+  return (
+    <>
+      <DeleteConfirmation
+        show={showModal}
+        cancel={() => setShowModal(false)}
+        approve={() => {
+          deletePolicyMutation.mutate(policyName);
+        }}
+        titleText={`Permanently remove the following policy ${policyName} ?`}
+      />
+
+      <Button
+        disabled
+        icon={<i className='fas fa-trash' />}
+        style={{ height: spacing.sp24, marginLeft: '0.6rem' }}
+        label=''
+        onClick={() => {
+          setShowModal(true);
+        }}
+        variant='danger'
+        tooltip={{ overlay: listPoliciesStatus === 'loading' ? 'Deleting...':'You cannot delete a predefined Scality Policy', placement: 'right' }}
+      />
+      {listPoliciesStatus === 'error' &&  <Banner
+        icon={<i className="fas fa-exclamation-triangle" />}
+        title="Error: Unable to delete policy"
+        variant="danger"
+      >
+        Error: Unable to delete policy.
+      </Banner>}
+    </>
+  );
+};
+
+const AccountPoliciesList = ({ accountName }: { accountName?: string }) => {
+  const history = useHistory();
+  const getQuery = (IAMClient: IAMClient) => getPoliciesQuery(notFalsyTypeGuard(accountName), IAMClient);
+  const getEntitiesFromResult = (data) => data.Policies;
+
+  const prepareData = (queryResult, search) => {
+    if (queryResult.firstPageStatus === 'success') {
+      const iamPolicies = queryResult && queryResult.data && queryResult.data.map((user) => {
+        return {
+          policyPath: user?.Path.substring(1),
+          policyName: user?.PolicyName,
+          modifiedOn: formatShortDate(user?.CreateDate),
+          attachments: user?.AttachmentCount,
+          arn: user?.Arn,
+          actions: null,
+        };
+      });
+
+      if (search) {
+        return iamPolicies.filter((user) =>
+          user.policyName?.toLowerCase().startsWith(search.toLowerCase()),
+        );
+      }
+
+      return iamPolicies;
+    }
+
+    return [];
+  };
+
+  const columns = [
+    {
+      Header: 'Policy Path',
+      accessor: 'policyPath',
+      cellStyle: {
+        minWidth: '10rem',
+      },
+    },
+    {
+      Header: 'Policy Name',
+      accessor: 'policyName',
+      cellStyle: {
+        minWidth: '18rem',
+      },
+      Cell({ value: policyName }: { value: string }) {
+          return (
+            <>
+              <Tooltip
+                overlay={'This is a Preseeded (predefined?) Scality Policy'}
```

Comment:
```suggestion
                overlay={'This is a predefined Scality Policy'}
```
And We are missing a condition here. We only display this tooltip for Scality Internal Policies.